src/popover.jsx

```diff
@@ -17,30 +17,34 @@ export default class Popover extends React.Component {
     this.state = {
       topOffset: 0,
       leftOffset: 0,
-      isOpen: false
+      isOpen: props.isOpen || false
     };
   }
 
-  componentDidMount() {
-    this.calculateHeights();
+  componentWillReceiveProps(props){
+    if(props.isOpen != this.state.isOpen) this.setState({isOpen: props.isOpen})
```

Comment:
Is this to manage the state in case you're managing it from say an AltContainer or something and it could receive new props at some point after initial mounting?